server/tests/src/test/java/org/infinispan/server/functional/HotRodAdminIT.java

```diff
@@ -0,0 +1,55 @@
+package org.infinispan.server.functional;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.exceptions.HotRodClientException;
+import org.infinispan.commons.configuration.XMLStringConfiguration;
+import org.infinispan.commons.test.Exceptions;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Ryan Emerson
+ * @since 12.0
+ */
+public class HotRodAdminIT {
```

Comment:
The test name shouldn't end in `IT`, because surefire discovers it as an integration test, tries to run it independently, and fails:
https://ci.infinispan.org/job/Infinispan/job/PR-8906/6/testReport/junit/org.infinispan.server.functional/HotRodCacheContinuousQueries/testQueries_true_/